birdhouse/components/stac/default.env

```diff
@@ -15,6 +15,14 @@ export STAC_BROWSER_VERSION=3.0.0-beta.5-crim-docker-image-push
 export STAC_BROWSER_IMAGE='ghcr.io/crim-ca/stac-browser:docker_image_push'
 export STAC_BROWSER_IMAGE_URI='${STAC_BROWSER_IMAGE}'
 
+export STAC_POPULATOR_BACKUP_DOCKER=ghcr.io/crim-ca/stac-populator
```

Comment:
Is this the same as `STAC_POPULATOR_{DOCKER,VERSION` in `optional-components/stac-populator/default.env`?  

Should we use the same version (default to the other variable `STAC_POPULATOR_{DOCKER,VERSION` if it is enabled) or it creates problems, so 2 separate version is better?

Reply:
Ah ok, I wasn't aware of the state of the stac-populator component.  Fine then.